libnavigator/src/main/java/com/mapbox/navigation/navigator/internal/NavigatorLoader.kt

```diff
@@ -111,9 +139,14 @@ object NavigatorLoader {
         }
     }
 
+    data class HistoryRecorders(
```

Comment:
It's not exactly recorders, is it? It's handles. It's confusing in `MapboxNavigation` because you have `historyRecorder`, `copilotHistoryRecorder` and `historyRecorders` while the last one is actually `historyRecorderHandles`.